ru/browser/browser/preferences/zen-preferences.ftl

```diff
@@ -35,41 +44,51 @@ zen-glance-trigger-shift-click =
 zen-glance-trigger-meta-click = 
     .label = Meta (Command) + Клик
 zen-glance-trigger-mantain-click = 
-    .label = Управлять Кликом
+    .label = Долгое нажатие (Скоро!)
+
 zen-look-and-feel-compact-view-header = Показать в компактном виде
 zen-look-and-feel-compact-view-description = Отображать только те панели, которые вы используете!
 zen-look-and-feel-compact-view-enabled = 
     .label = Включить компактный режим { -brand-short-name }
+
 zen-look-and-feel-compact-view-top-toolbar = 
-    .label = Скрыть верхнюю панель инструментов и в компактном режиме
+    .label = Скрывать в компактном режиме также и верхнюю панель
+
 zen-look-and-feel-compact-toolbar-flash-popup = 
-    .label = Кратко сделать всплывающее окно панели инструментов при переключении или открытии новых вкладок в компактном режиме
-zen-look-and-feel-buttons-header = Стиль кнопки
-zen-look-and-feel-buttons-description = Выберите стиль кнопок, которые вы хотите увидеть
+    .label = Показать на короткое время всплывающее окно панели инструментов при переключении или открытии новых вкладок в компактном режиме
```

Comment:
Despite string's name saying the opposite, this preference would affect the tab side-pane and not the search bar (+ extensions and instruments). But this is an issue to correct the strings in general, I guess. Gotta ask @mauro-balades

 Also, let's keep it a bit less verbose.

```suggestion
    .label = Кратковременно показывать панель инструментов при действиях с вкладками
```